src/main/java/uk/nhs/digital/nhsconnect/nhais/translator/amendment/mappers/AmendmentAddressToEdifactMapper.java

```diff
@@ -0,0 +1,135 @@
+package uk.nhs.digital.nhsconnect.nhais.translator.amendment.mappers;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import lombok.RequiredArgsConstructor;
+import uk.nhs.digital.nhsconnect.nhais.exceptions.FhirValidationException;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.PersonAddress;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.Segment;
+import uk.nhs.digital.nhsconnect.nhais.model.jsonpatch.AmendmentBody;
+import uk.nhs.digital.nhsconnect.nhais.model.jsonpatch.AmendmentPatch;
+import uk.nhs.digital.nhsconnect.nhais.model.jsonpatch.AmendmentPatchOperation;
+import uk.nhs.digital.nhsconnect.nhais.model.jsonpatch.AmendmentValue;
+import uk.nhs.digital.nhsconnect.nhais.model.jsonpatch.JsonPatches;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class AmendmentAddressToEdifactMapper extends AmendmentToEdifactMapper {
+
+    @Override
+    Segment mapPatches(AmendmentBody amendmentBody) {
+        var patches = amendmentBody.getJsonPatches();
+
+        var houseName = patches.getHouseName()
```

Comment:
So this will be the actual value or an empty string (because of what `getNullableFormattedSimpleValue` does) or null? Why the empty string then?

Reply:
If it's not in `patches` then `getHouseName()` will return empty Optional, so mapping won't take place